### ISSUE TITLE
L1 loss on validation data

### ISSUE THREAD

**Issue by heidotrofimov** [affinelayer/pix2pix-tensorflow#200]
I modified the tensorflow 2 version of the code so that it now computes two L1 losses while training, one on training data and other on validation data. The code can be accessed here: https://github.com/affinelayer/pix2pix-tensorflow/pull/199

Would be nice if someone reviews the code and affirms that it does the right thing (I believe it does). It will use the generator twice, once on training data and once on validation data, and will compute accordingly two different L1 losses. Only the training data results are used in subsequent model training.
